common/services/prismic/pages.js

```diff
@@ -41,8 +41,12 @@ export function parsePage(document: PrismicDocument): Page {
     return parseSeason(season);
   });
   const parentPages = parseSingleLevelGroup(data.parents, 'parent').map(
-    page => {
-      return parsePage(page);
+    (page, index) => {
+      return {
+        ...parsePage(page),
+        order: data.parents[index].order,
```

Comment:
What's the difference between `page` and `data.parents[index]`?

Reply:
Nothing. Of course.